### PR TITLE
bug: Dynamic Balance 

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
makes the balance on top left of the sidebar menu dynamic.

### Issues
Is there one?

### CODE REVIEW

**Comment by einhornimmond** [comment]
You hasn't merged my second commit to community_bug_fix_27_04_2021

File: frontend/src/views/Layout/DashboardLayout_gdd.vue [hunk 1]
```diff
@@ -78,10 +84,24 @@ export default {
       this.$store.dispatch('logout')
       this.$router.push('/login')
     },
+    async loadBalance() {
+      const result = await communityAPI.balance(this.$store.state.session_id)
+      if (result.success) {
+        this.balance = result.result.data.balance / 10000
+      } else {
+        // what to do when loading balance fails?
```
Comment (ulfgebhardt) [inline frontend/src/views/Layout/DashboardLayout_gdd.vue:92]: toast + 0 ? toast + NaN/NA?

**Review by ulfgebhardt — COMMENTED** [review]
Looks quite good already

File: frontend/src/views/KontoOverview/GddSend.vue [hunk 2]
```diff
@@ -256,27 +257,29 @@ export default {
...
       )
-      // console.log(result)
-
       if (result.success) {
-        // console.log('send success')
-
-        this.$emit('change-rows', { row_form: false, row_check: false, row_thx: true })
+        this.$emit('toggle-show-list', false)
+        this.row_check = false
+        this.row_thx = true
```
[the remark this answers is not in the document]
Comment (Mogge) [reply frontend/src/views/KontoOverview/GddSend.vue:263]: row_check displays the row to confirm the transaction.
row_thx displays the success of the transaction

**Comment by einhornimmond** [comment]
I have now merged #261 into this. 
It is already in master, but now also here. 
Because skeema didn't work out of the box, you need to start skeema manually again, after mariadb is online. 
After skeema has updated you db structure you need to manually update content in login-server db.
Simply copy into sql section in phpmyadmin:
update groups set host = "nginx";

File: frontend/src/views/Layout/DashboardLayout_gdd.vue (hunk 1 above)
Comment (Mogge) [reply frontend/src/views/Layout/DashboardLayout_gdd.vue:92]: balance is 0 by default. For now I would like to assume that this cannot fail, and touch it again, when we have a proper toaster.

Comment (ulfgebhardt) [reply frontend/src/views/Layout/DashboardLayout_gdd.vue:92]: Issue pls

File: frontend/src/views/KontoOverview/GddSend.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (ulfgebhardt) [reply frontend/src/views/KontoOverview/GddSend.vue:265]: Isn't that an exclusive process like step 1,2,3. Now you can have both active?! Naming is also not self explanatory

File: frontend/src/views/KontoOverview/GddSend.vue [hunk 3]
```diff
@@ -226,6 +225,8 @@ export default {
         auto_sign: true,
       },
       send: false,
+      row_check: false,
+      row_thx: false,
```
Comment (ulfgebhardt) [inline frontend/src/views/KontoOverview/GddSend.vue:229]: find das echt semi, sollte das nicht eine variable sein, die den Prozess abbildet?  `current_send_step = 0`. Warum stehen alle auf `false`

**Review by ulfgebhardt — CHANGES_REQUESTED** [review]

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**